### ISSUE TITLE
Crash because megamix path cannot be found

### ISSUE THREAD

**Issue by RhythmLunatic** [Brod8362/d4m#28]
I installed Mega Mix on a second steam library path, but it seems like it wouldn't really work with the default either
```
Traceback (most recent call last):
  File "/home/gaming-linux/.local/bin/d4m-gui", line 8, in <module>
    sys.exit(main())
  File "/home/gaming-linux/.local/lib/python3.10/site-packages/d4m/gui.py", line 652, in main
    if not d4m.common.modloader_is_installed(megamix_path):
  File "/home/gaming-linux/.local/lib/python3.10/site-packages/d4m/common.py", line 44, in modloader_is_installed
    return os.path.isfile(os.path.join(megamix_path, "config.toml"))
  File "/usr/lib/python3.10/posixpath.py", line 76, in join
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```
D4M_INSTALL_DIR variable works, but I think it would be better if it printed a descriptive error telling you to use it if it can't find the install dir.

**Comment by Brod8362** [comment]
Thanks for the bug report. What version of d4m are you running, and what is your megamix install path?

**Comment by Brod8362** [comment]
Fixed in v0.3.3.

**Comment by RhythmLunatic** [comment]
Sorry for the late reply, I installed d4m from pip and my megamix install path was "/media/WD_Black/SteamLibrary/steamapps/Project Diva MegaMix+" or something similar. But since it worked fine with D4M_INSTALL_DIR I just added it to .bashrc
